pkg/v1/service/books/storage.go

```diff
@@ -0,0 +1,102 @@
+package books
+
+import (
+	"database/sql"
+	"log"
+
+	_ "github.com/mattn/go-sqlite3"
```

Comment:
The underscore isn't needed if you aren't going to override the default package name.

Reply:
This was needed to install the correct DB driver. Added a comment.